lib/util/devutil.js

```diff
@@ -159,10 +160,15 @@ devutil.makeIdentity = function(serial, properties) {
     model = model.substr(manufacturer.length)
   }
 
-  if (marketName) {
-    var devices = androidDeviceList.getDevicesByDeviceId(marketName)
-    if (devices.length > 0) {
-      marketName = devices[0].name
+  if (customMarketName) {
+    marketName = customMarketName
+  }
+  else {
+    if (marketName) {
```

Comment:
Those 2 nesting levels can be merged to a single `else if`

Reply:
hmm okay thats right 👍  will do it

Reply:
okay done 👍